File: 1º Semestre/Programação de Computadores/Lista de Exercicios 2/12.cpp
#include <iostream>
#include <string>
using namespace std;

int main(){
	float numero;
	
	printf("Digite um numero: ");
	scanf("%f", &numero);
	
	if(numero >= 20 || numero <= 90){
		printf("%0.0f esta entre 20 e 90", numero);
	}
}

File: 1º Semestre/Programação de Computadores/Lista de Exercicios 2/Python/12.py
"""
12. Elabore um algoritmo que indique se um número digitado está compreendido entre 20 e
90, ou não
"""

numInput = float(input("Digite um numero: "))

if numInput >= 20 and numInput <= 90:
    print("O numero {:.2f} está entre 20 e 90.".format(numInput))
else:
    print("O numero {:.2f} não está entre 20 e 90.".format(numInput))